src/lib/locales/en.json

```
@@ -160,9 +160,11 @@
 			"addInvestment": "Add investment",
 			"editInvestment": "Edit investment",
 			"investmentName": "Investment name",
-			"importDataInvestment": "Import data for this investment",
+			"importDataInvestment": "Import data for this investmen",
 			"importData": "Import data",
-			"isin": "ISIN",
+			"isin": "Please enter an ISIN number or a ticker symbol",
+			"isinInputPlaceholder": "US0378331005 or AAPL",
```

Feel free to reject this, I just find it clearer
```suggestion
			"isinInputPlaceholder": "e.g. US0378331005 or AAPL",
```